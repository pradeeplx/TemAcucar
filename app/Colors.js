export default Colors = {
  brown: '#654C50',
  pink: '#FF7F85',
  lightPink: '#FFF2F3',
  mediumLightPink: '#FEBCC3',
  mediumPink: '#FC9CA3',
  white: '#FFFFFF',
  black: '#000000',
  beige: '#E6D3B7',
  darkBeige: '#DBC3A3',
  lightBeige: '#FAF6F1',
  mediumLightBeige: '#FFE9AE',
  yellow: '#FFEA2D',
  lightYellow: '#FFFBD5',
  darkYellow: '#FFBA00',
  blue: '#44B5B1',
  lightBlue: '#ECF8F7',
  gray: '#CECED2',
  lightGray: '#EFEDED',
  darkGray: '#807F7F',
  facebook: '#557BD3',
  red: '#E9122B',
  green: '#2CB36E',
}
