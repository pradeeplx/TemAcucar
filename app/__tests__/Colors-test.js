import 'react-native'
import React from 'react'
import Colors from '../Colors'

test('Colors object', () => {
  expect(Colors).toMatchSnapshot()
})
