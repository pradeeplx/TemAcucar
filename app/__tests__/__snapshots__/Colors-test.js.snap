exports[`test Colors object 1`] = `
Object {
  "beige": "#E6D3B7",
  "black": "#000000",
  "blue": "#44B5B1",
  "brown": "#654C50",
  "darkBeige": "#DBC3A3",
  "darkGray": "#807F7F",
  "darkYellow": "#FFBA00",
  "facebook": "#557BD3",
  "gray": "#CECED2",
  "green": "#2CB36E",
  "lightBeige": "#FAF6F1",
  "lightBlue": "#ECF8F7",
  "lightGray": "#EFEDED",
  "lightPink": "#FFF2F3",
  "lightYellow": "#FFFBD5",
  "mediumLightBeige": "#FFE9AE",
  "mediumLightPink": "#FEBCC3",
  "mediumPink": "#FC9CA3",
  "pink": "#FF7F85",
  "red": "#E9122B",
  "white": "#FFFFFF",
  "yellow": "#FFEA2D",
}
`;
